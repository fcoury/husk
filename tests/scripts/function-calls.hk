fn func2() {
    println!("Function 2 called");
    func1();
}

fn func1() {
    println!("Function 1 called");
}

func2();
