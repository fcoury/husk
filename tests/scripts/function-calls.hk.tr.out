Function 2 called
Function 1 called
